aspnetcore/performance/caching/distributed.md

```diff
@@ -23,7 +23,7 @@ When cached data is distributed, the data:
 
 Distributed cache configuration is implementation specific. This article describes how to configure SQL Server and Redis distributed caches. Third party implementations are also available, such as [NCache](http://www.alachisoft.com/ncache/aspnet-core-idistributedcache-ncache.html) ([NCache on GitHub](https://github.com/Alachisoft/NCache)). Regardless of which implementation is selected, the app interacts with the cache using the <xref:Microsoft.Extensions.Caching.Distributed.IDistributedCache> interface.
 
-[View or download sample code](https://github.com/aspnet/Docs/tree/master/aspnetcore/performance/caching/distributed/sample) ([how to download](xref:tutorials/index#how-to-download-a-sample))
+[View or download sample code](https://github.com/aspnet/Docs/tree/master/aspnetcore/performance/caching/distributed/samples/2.x/DistCacheSample) ([how to download](xref:tutorials/index#how-to-download-a-sample))
```

Comment:
Point this to the `samples` folder because we want the versions to show for the sample. That way, we don't need specific version links (e.g., 1.x, 2.x, 3.x) in the topic. We just shoot the reader to `samples` and then they'll see `1.x`, `2.x`, and later `3.x` folders.